files/text-file.conf

```diff
@@ -24,12 +25,10 @@ input(type="imfile"
 
 # rsyslog RuleSets
 ruleset(name="FILE_TAG_roleset") {
-    if $programname == "FILE_TAG" then {
```

Comment:
Only the file input module supports the TAG descriptor. If you use tcp / udp inputs, its not a valid descriptor so any events sent couldn't be configured to match the condition for forwarding. 

In the docker image its still requiring a manual configuration for ENV MONITOR_FILE_TYPE so it can and does still use that to send the log type unless I'm misunderstanding?

Reply:
The condition was just to make sure the file type match the program name,
but since the user dose configure the file type and path i guess it can be dismissed